subworkflows/nf-core/vcf_joint_calling_germline_gatk/main.nf

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env nextflow
+
+include { GATK4_GENOMICSDBIMPORT                                } from '../../../modules/nf-core/gatk4/genomicsdbimport/main'
+include { GATK4_GENOTYPEGVCFS                                   } from '../../../modules/nf-core/gatk4/genotypegvcfs/main'
+include { GATK4_MERGEVCFS as MERGE_GENOTYPEGVCFS                } from '../../../modules/nf-core/gatk4/mergevcfs/main'
+include { TABIX_TABIX as TABIX                                  } from '../../../modules/nf-core/tabix/tabix/main'
+include { BEDTOOLS_MULTIINTER                                   } from '../../../modules/nf-core/bedtools/multiinter/main'
+
+include { BED_SCATTER_BEDTOOLS as BED_SCATTER                   } from '../../subworkflows/nf-core/bed_scatter_bedtools/main'
+
+workflow VCF_JOINT_CALLING_GERMLINE_GATK {
+
+    take:
+    ch_gvcfs            // channel: [ val(meta), [ gvcf ], [ gvcf_index ]]
+    ch_beds             // channel: [ val(meta), [ bed ]]
+    ch_fasta            // channel: [ val(meta), /path/to/reference/fasta]
+    ch_fai              // channel: [ val(meta), /path/to/reference/fasta/index]
+    ch_dict             // channel: [ val(meta), /path/to/reference/fasta/dict]
+    ch_dbsnp            // channel: [ val(meta), /path/to/dbsnp/vcf]
+    ch_dbsnp_tbi        // channel: [ val(meta), /path/to/dbsnp/vcf/index]
+    scatter_count       // channel: val(scatter_count)
+
+    main:
+    ch_versions    = Channel.empty()
+
+    // Merge bed files to create a single interval list
+    // BEDTOOLS_MULTIINTER( [meta, bed], chrom_sizes )
+    BEDTOOLS_MULTIINTER( ch_beds, [] )
+    ch_versions = ch_versions.mix(BEDTOOLS_MULTIINTER.out.versions).first()
+
+    // Split merged bed into individual intervals for genotyping
+    // BED_SCATTER_BEDTOOLS( [meta, bed, scatter_count] )
+    BED_SCATTER_BEDTOOLS(BEDTOOLS_MULTIINTER.out.bed, scatter_count)
```

Comment:
I think you can leave the scatter/gather out of the subworkflow too as what's being discussed here: https://github.com/nf-core/modules/pull/2427#discussion_r1040655133

Reply:
This way it's easier for the pipeline developer to just mix and match the subworkflows they prefer instead of making several subwfs with each method

Reply:
I think the whole point of this subwf is to make it easier to have parallel joint GT in a pipeline, same for the haplotyper one.

By including the scatter/gather logic, a dev can import the subwf and be confident it runs in a parallel way without having to program any logic for it

Reply:
if we keep reducing functionality to the bare minimum we just end up with modules again IMO 🤷🏻

Reply:
The scatter/gather subworkflows I made don't actually need any more channel logic to fully use. You can just run them right after each other. But it's something that's still in discussion, we should probably discuss it with some more people. (I think designing the subworkflows so they work correctly with each other but keep them separate is the most flexible solution without giving up the ease of subworkflows)

Reply:
I'll make a gather subwf with mergevcfs in the near future too :)

Reply:
> if we keep reducing functionality to the bare minimum we just end up with modules again IMO 🤷🏻

Yep that's why we should probably fully discuss this first